[Bug]: Cannot find module '/node_modules/@dodopayments/better-auth/dist/hooks/customer'  imported from /node_modules/@dodopayments/better-auth/dist/index.js

### Pre-submission Checklist

- [x] I have searched existing issues to ensure this is not a duplicate
- [x] I have read the documentation
- [x] I am using the latest version of the affected package

### Affected Package

@dodopayments/betterauth

### Package Version

1.2.0

### Bug Severity

Critical - Blocks core functionality

### Bug Description

### Error
```
Cannot find module '/node_modules/@dodopayments/better-auth/dist/hooks/customer' 
imported from /node_modules/@dodopayments/better-auth/dist/index.js
```

### Steps to Reproduce

### Reproduction
https://stackblitz.com/edit/github-e6haflig?file=app%2Fapp.vue

Simple Nuxt 4 app with the modules shown in this docs page installed:
https://docs.dodopayments.com/developer-resources/better-auth-adaptor

And an import of dodopayments, checkout, portal, webhooks modules in app.vue
`import {
  dodopayments,
  checkout,
  portal,
  webhooks,
} from '@dodopayments/better-auth';`

### Expected Behavior

The package @dodopayments/better-auth should successfully build and import all required modules when used in a Nuxt 4 application. Importing from the package (e.g. import { customer } from '@dodopayments/better-auth') should work without module resolution errors, as documented.

### Actual Behavior

Cannot find module '/node_modules/@dodopayments/better-auth/dist/hooks/customer'
imported from /node_modules/@dodopayments/better-auth/dist/index.js


### Code Sample

import {
  dodopayments,
  checkout,
  portal,
  webhooks,
} from '@dodopayments/better-auth';

### Error Logs

_No response_

### Environment Information

### Environment
- Nuxt: v4.1.3
- Node: v20.19.1

### Additional Context

### Dependencies
```json
{
        "@dodopayments/better-auth": "^1.2.0",
        "better-auth": "^1.3.28",
        "dodopayments": "^2.2.1",
        "nuxt": "^4.1.3",
        "vue": "^3.5.22",
        "vue-router": "^4.5.1",
        "zod": "^4.1.12"
      }
```

### Contributing

- [ ] I would be willing to submit a PR to fix this issue

--- a/packages/betterauth/package.json
+++ b/packages/betterauth/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@dodopayments/better-auth",
-  "version": "1.3.2",
+  "version": "1.3.3",
   "publishConfig": {
     "access": "public"
   },
@@ -18,7 +18,7 @@
   },
   "types": "./dist/index.d.ts",
   "dependencies": {
-    "@dodopayments/core": "^0.3.1"
+    "@dodopayments/core": "^0.3.2"
   },
   "engines": {
     "node": ">=16.0.0"

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -1,31 +1,37 @@
 {
   "name": "@dodopayments/core",
-  "version": "0.3.1",
+  "version": "0.3.2",
+  "type": "module",
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
-    "build": "tsc",
-    "dev": "tsc --watch",
+    "build": "tsup src/ --format esm,cjs --dist --clean --sourcemap",
+    "dev": "npm run build -- --watch",
     "test": "vitest --run",
     "test:watch": "vitest --watch",
     "check-types": "tsc --noEmit"
   },
   "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
     "./checkout": {
+      "types": "./dist/checkout/checkout.d.ts",
       "import": "./dist/checkout/checkout.js",
-      "require": "./dist/checkout/checkout.cjs",
-      "types": "./dist/checkout/checkout.d.ts"
+      "require": "./dist/checkout/checkout.cjs"
     },
     "./webhook": {
+      "types": "./dist/webhook/webhook.d.ts",
       "import": "./dist/webhook/webhook.js",
-      "require": "./dist/webhook/webhook.cjs",
-      "types": "./dist/webhook/webhook.d.ts"
+      "require": "./dist/webhook/webhook.cjs"
     },
     "./schemas": {
+      "types": "./dist/schemas/webhook.d.ts",
       "import": "./dist/schemas/webhook.js",
-      "require": "./dist/schemas/webhook.cjs",
-      "types": "./dist/schemas/webhook.d.ts"
+      "require": "./dist/schemas/webhook.cjs"
     }
   },
   "dependencies": {

--- a/packages/core/src/index.ts
+++ b/packages/core/src/index.ts
@@ -1,0 +1,77 @@
+export {
+  checkoutQuerySchema,
+  dynamicCheckoutBodySchema,
+  checkoutSessionProductCartItemSchema,
+  checkoutSessionCustomerSchema,
+  checkoutSessionBillingAddressSchema,
+  paymentMethodTypeSchema,
+  checkoutSessionCustomizationSchema,
+  checkoutSessionFeatureFlagsSchema,
+  checkoutSessionSubscriptionDataSchema,
+  checkoutSessionPayloadSchema,
+  checkoutSessionResponseSchema,
+  createCheckoutSession,
+  buildCheckoutUrl,
+} from "./checkout/checkout";
+
+export type {
+  CheckoutHandlerConfig,
+  CheckoutSessionPayload,
+  CheckoutSessionResponse,
+  CheckoutSessionProductCartItem,
+  CheckoutSessionCustomer,
+  CheckoutSessionBillingAddress,
+  CheckoutSessionCustomization,
+  CheckoutSessionFeatureFlags,
+  CheckoutSessionSubscriptionData,
+  PaymentMethodType,
+  CheckoutSessionHandlerConfig,
+} from "./checkout/checkout";
+
+export {
+  PaymentSchema,
+  SubscriptionSchema,
+  RefundSchema,
+  DisputeSchema,
+  LicenseKeySchema,
+  PaymentSucceededPayloadSchema,
+  PaymentFailedPayloadSchema,
+  PaymentProcessingPayloadSchema,
+  PaymentCancelledPayloadSchema,
+  RefundSucceededPayloadSchema,
+  RefundFailedPayloadSchema,
+  DisputeOpenedPayloadSchema,
+  DisputeExpiredPayloadSchema,
+  DisputeAcceptedPayloadSchema,
+  DisputeCancelledPayloadSchema,
+  DisputeChallengedPayloadSchema,
+  DisputeWonPayloadSchema,
+  DisputeLostPayloadSchema,
+  SubscriptionActivePayloadSchema,
+  SubscriptionOnHoldPayloadSchema,
+  SubscriptionRenewedPayloadSchema,
+  SubscriptionPausedPayloadSchema,
+  SubscriptionPlanChangedPayloadSchema,
+  SubscriptionCancelledPayloadSchema,
+  SubscriptionFailedPayloadSchema,
+  SubscriptionExpiredPayloadSchema,
+  LicenseKeyCreatedPayloadSchema,
+  WebhookPayloadSchema,
+} from "./schemas/webhook";
+
+export type {
+  Payment,
+  Subscription,
+  Refund,
+  Dispute,
+  LicenseKey,
+  WebhookPayload,
+  HandlerWithContext,
+  WebhookEventHandlers,
+} from "./schemas/webhook";
+
+export {
+  type WebhookHandlerConfig,
+  handleWebhookPayload,
+  verifyWebhookPayload,
+} from "./webhook/webhook";

